packages/scss/src/commons/base.scss

```diff
@@ -147,6 +155,12 @@
 			}
 		}
 
+		button {
```

Comment:
pas possible de mettre la class "menu-list-item-action" ?

Reply:
That was our first idea, but the focus style is applied by default on every `<a>`, so it seemed more consistent to do the same thing on `<button>` (and we're not taking huge risks by automatically replacing default outline browser by ours.)